proxy/pkg/zdmproxy/clusterconn.go

```diff
@@ -361,7 +390,16 @@ func (cc *ClusterConnector) handleAsyncResponse(response *frame.RawFrame) *frame
 }
 
 func (cc *ClusterConnector) sendRequestToCluster(frame *frame.RawFrame) {
-	cc.writeCoalescer.Enqueue(frame)
+	var err error
+	if cc.frameProcessor != nil {
+		frame, err = cc.frameProcessor.AssignUniqueId(frame)
+	}
+	if err != nil {
+		log.Errorf("couldn't assign unique id to frame %v", frame)
```

Comment:
`log.Errorf("[%v] Couldn't assign stream id to frame %v: %v, frame.Header.OpCode, err)`

Reply:
Done